wicket-core/src/main/java/org/apache/wicket/protocol/http/WebApplication.java

```diff
@@ -758,6 +759,14 @@ protected void internalInit()
 		setAjaxRequestTargetProvider(AjaxRequestHandler::new);
 
 		getAjaxRequestTargetListeners().add(new AjaxEnclosureListener());
+		
+		getHeaderContributorListeners().add(head -> {
+			Optional<CssResourceReference> wicketCoreCSS = getResourceSettings().getWicketCoreCSS();
```

Comment:
personal preference: use `Optional.ifPresent()`

Reply:
I usually do not want to use lambda's inside lambda's, that's why I did it like this, but I rewrote it with `ifPresent` and it does look better that way. I'll change the code.